internal/util/util.go

```diff
@@ -37,23 +39,76 @@ func BundleInstanceProvisionerFilter(provisionerClassName string) predicate.Pred
 
 func MapBundleToBundleInstanceHandler(cl client.Client, log logr.Logger) handler.MapFunc {
 	return func(object client.Object) []reconcile.Request {
-		b := object.(*rukpakv1alpha1.Bundle)
+		b, ok := object.(*rukpakv1alpha1.Bundle)
+		if !ok {
+			return nil
+		}
```

Comment:
Did you run into a problem here? I wonder if we should let this panic to warn us that we're using this in an unintended way.

Reply:
Nope - I can revert this change.